drivers/sensor/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ add_subdirectory_ifdef(CONFIG_TI_HDC20XX	ti_hdc20xx)
 add_subdirectory_ifdef(CONFIG_HMC5883L		hmc5883l)
 add_subdirectory_ifdef(CONFIG_HP206C		hp206c)
 add_subdirectory_ifdef(CONFIG_HTS221		hts221)
+add_subdirectory_ifdef(CONFIG_I3G4250D      i3g4250d)
```

Comment:
Replace the spaces with tabs to be consistent with the rest of the file

Reply:
Done.